mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -50,6 +50,13 @@ void CGameModeBase::SetGameModeVars()
     sv_edge_fix.SetValue(false); // MOM_TODO Let people test the edge fix in 0.8.4 so we can get their opinions
 }
 
+float CGameModeBase::GetJumpFactor()
+{
+    // height == 57 on 800 gravity
+    // x = sqrt(height * 2 * gravBase) = sqrt(57 * 2 * 800) ~ 301.99337741083
+    return 301.99337741083f;
```

Comment:
Would be nicer to just write sqrt(2.0f * 800.0f * 57.0f) instead as the value should be evaluated at compile time.

Reply:
It would make more sense yeah, this is just an optimization to avoid having to square root & do other calculations.

Reply:
I think the compiler is smart enough to do the calculation when compiling.

Reply:
You're right :)